content/desktop/synchronized-file-sharing.md

```diff
@@ -0,0 +1,101 @@
+
+After creating a file share instance, any container using a bind mount that references a host filesystem location which matches the stated synchronized file share host location, is provided by the Synchronized file sharing feature. Bind mounts that don't satisfy this condition are passed to the normal virtual filesystem bind-mounting mechanism, for example VirtioFS or gRPC-FUSE.
+
+> **Important**
+>
+> Synchronized file sharing isn't available on WSL or when using Windows containers. 
+{ .important }
+
+## Set up
+
+Before you get started with Synchronized file sharing, you need to:
+
+- Download and install [Docker Desktop version 4.25](release-notes.md)
+- Have a Pro, Team, or Business subscription
+- Have a Docker ID that's part of a Docker organization participating in the Early access program
+
+To enable Synchronized file sharing:
+1. Sign in to Docker Desktop.
+2. Anywhere within the Docker Dashboard, enter the Konami Code which has the key sequence of `up, up, down, down, left, right, left, b, a`.
+3. From the list of experimental features, select **Mutagen File Sharing**.
+
+You can now create your first file share instance.
+
+## Create a file share instance 
+
+To create a synchronized file share:
+1. Navigate to **Settings** within Docker Desktop and then within the **Resource** section, select the **File sharing** tab. 
+2. In the **Synchronized file shares** section, select the **plus** icon.
+3. Select a host folder to share. The synchronized file share should initialize and be usable.
+
+File shares take a few seconds to initialize as files are copied into the Docker Desktop VM. During this time, the status indicator displays **Preparing**.
+
+When the status indicator displays **Watching for filesystem changes**, your files are available to the VM through all the standard bind mount mechanisms, whether that's `-vm` in the command line or specified in your `compose.yml` file.
+
+## Explore your file share instance
+
+The **Synchronized file shares** section displays all your file share instances and provides useful information about each instance including:
+- The origin of the file share content
+- A Status update
+- How much space each file share is using
+- The number of filesystem entry counts
+- The number of symbolic links
+- Which container(s) is using the file share instance
+
+Selecting a file share instance expands the dropdown and exposes this information.
+
+## Use `.syncignore`
+
+You can use a `.syncignore` file at the root of each file share, to exclude local files from your file share instance. It supports the same syntax as `.dockerignore` files and excludes, and/or re-includes, paths from synchronization. `.syncignore` files are ignored at any location other than the root of the file share.
+ 
+Some example of things you might want to add to your `.syncignore` file are:
+- ?
+
+In general, the contents of your `.syncignore` file should be similar to what you have in your `.gitignore` file.
```

Comment:
I'm not sure if this is an ideal heuristic for describing the `.syncignore` contents.  There are some fairly significant semantic differences between `.dockerignore`/`.syncignore` and `.gitignore` ignore syntaxes, so I wouldn't want users to copy/paste and expect it to work.  Less importantly: the content ignored during the inner loop may not align entirely with what goes into `.gitignore`.  Maybe we could just add a caveat about the syntactic differences... I'm not sure the best way to approach it.

Reply:
I've adjusted this sentence to reflect your comment above. I actually think this is fine for now, we can revisit this post-EAP and monitor if users need extra clarification